exercises/practice/clock/ClockTest.php

```diff
@@ -31,204 +9,527 @@ public static function setUpBeforeClass(): void
+     * @testdox subtract minutes
+     */
+    public function testSubtractMinutes(): void
+    {
+        $clock = new Clock(10, 3);
+        $clock = $clock->sub(3);
+        $this->assertEquals("10:00", $clock);
+    }
 
-        $this->assertEquals('23:30', $clock->__toString());
+    /**
+     * uuid 0aafa4d0-3b5f-4b12-b3af-e3a9e09c047b
+     * @testdox subtract to previous hour
+     */
+    public function testSubtractToPreviousHour(): void
+    {
+        $clock = new Clock(10, 3);
+        $clock = $clock->sub(30);
+        $this->assertEquals("09:33", $clock);
     }
 
-    public function testWrapAroundDay(): void
+    /**
+     * uuid 9b4e809c-612f-4b15-aae0-1df0acb801b9
+     * @testdox subtract more than an hour
+     */
+    public function testSubtractMoreThanAnHour(): void
     {
-        $clock = new Clock(5, 32);
+        $clock = new Clock(10, 3);
+        $clock = $clock->sub(70);
+        $this->assertEquals("08:53", $clock);
+    }
 
-        $clock = $clock->add(25 * 60);
+    /**
+     * uuid 8b04bb6a-3d33-4e6c-8de9-f5de6d2c70d6
+     * @testdox subtract across midnight
+     */
+    public function testSubtractAcrossMidnight(): void
+    {
+        $clock = new Clock(0, 3);
+        $clock = $clock->sub(4);
+        $this->assertEquals("23:59", $clock);
+    }
 
-        $this->assertEquals('06:32', $clock->__toString());
+    /**
+     * uuid 07c3bbf7-ce4d-4658-86e8-4a77b7a5ccd9
+     * @testdox subtract more than two hours
+     */
+    public function testSubtractMoreThanTwoHours(): void
+    {
+        $clock = new Clock(0, 0);
+        $clock = $clock->sub(160);
+        $this->assertEquals("21:20", $clock);
     }
 
-    public function testWrapAroundDayBackwards(): void
+    /**
+     * uuid 90ac8a1b-761c-4342-9c9c-cdc3ed5db097
+     * @testdox subtract more than two hours with borrow
+     */
+    public function testSubtractMoreThanTwoHoursWithBorrow(): void
     {
-        $clock = new Clock(5, 32);
+        $clock = new Clock(6, 15);
+        $clock = $clock->sub(160);
+        $this->assertEquals("03:35", $clock);
+    }
 
-        $clock = $clock->sub(25 * 60);
+    /**
+     * uuid 2149f985-7136-44ad-9b29-ec023a97a2b7
+     * @testdox subtract more than one day (1500 min = 25 hrs)
+     */
+    public function testSubtractMoreThanOneDay(): void
+    {
+        $clock = new Clock(5, 32);
+        $clock = $clock->sub(1500);
+        $this->assertEquals("04:32", $clock);
+    }
 
-        $this->assertEquals('04:32', $clock->__toString());
+    /**
+     * uuid ba11dbf0-ac27-4acb-ada9-3b853ec08c97
+     * @testdox subtract more than two days
+     */
+    public function testSubtractMoreThanTwoDays(): void
+    {
+        $clock = new Clock(2, 20);
+        $clock = $clock->sub(3000);
+        $this->assertEquals("00:20", $clock);
     }
 
-    public function testEquivalentClocks(): void
+    /**
+     * uuid f2fdad51-499f-4c9b-a791-b28c9282e311
+     * @testdox clocks with same time
```

Comment:
And the last outer `cases` change:

```suggestion
     * @testdox Compare two clocks for equality - clocks with same time
```